src/InputWithLabel.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react'
+import { useRef, useEffect } from 'react'
+
+function InputWithLabel({ todoTitle, handleTitleChange, children }) {
+	const inputRef = useRef()
+
+	useEffect(() => {
+		inputRef.current.focus()
+	})
+	return (
+		<>
+			<label htmlFor='todoTitle'>{children}</label>
```

Comment:
`label` and `input` are not connected. Maintain abstract naming convention when deciding what name to go with.